style.css

```diff
@@ -118,6 +122,15 @@ footer {
   box-shadow: 0 1px 24px 1px rgba(0, 0, 0, 0.48);
 }
 
+.gameBoard > .top {
```

Comment:
Played around with this on my machine, and it might make more sense if we made the *top* div slightly less wide? Since the elements are spaced evenly, I think it looks a little off if the title and logo stretch outside of the oval for the game board. I added margin left and right of 20%, and it seemed to bring these items back into the board. Just a thought!

Reply:
Yeah, makes sense! I'll merge and add another small change.